docs/rbac.rst

```diff
@@ -0,0 +1,269 @@
+Role Based Access Control
+=========================
+
+.. important::
+
+    This feature is a Tech Preview and can be subject of change.
+
+.. important::
+
+    Using user ``admin`` will bypass all RBAC checks.
```

Comment:
Does it actually bypass the checks or does the admin simply possess every permission?  I thought it was the latter https://github.com/pulp/pulp_rpm/pull/2418/files#diff-37aef788d3a12e65873c374ff05bc8629a8b636cab51ac31c127f94d43f1314eR234

The end result is the same but we should properly communicate what's going on :)

Reply:
rephrased to "admin gets all permissions"

Reply:
The admin is assumed to have all the permissions (even before asking any permission backend). But the access policy can still disallow _any_ access. So there is actually a difference in the end result.